cli/report/report.py

```diff
@@ -215,8 +253,41 @@ def file_jira_issues(
                 )
                 bugs_filed.append(jira_issue.key)
 
+        self._bugs_filed += bugs_filed
         return bugs_filed
 
+    @staticmethod
+    def create_jira_issue(
```

Comment:
I'm a little bit confused as to why this was added rather than just directly calling `firewatch_config.jira.create_issue` as it originally was.

Reply:
This change made it simpler to patch and inspect the inputs used for creating new Jira issues. I also felt like this was closer to the level of abstraction used for the [`add_duplicate_comment`](https://github.com/CSPI-QE/firewatch/blob/878610b2dba34b22e0d3b249b9891bf336bc86fc/cli/report/report.py#L445) method.

Reply:
[This test](https://github.com/amp-rh/firewatch/blob/fd0b141866f2cb1df3f0a9b54418a0b0f7e79a6b/tests/unittests/objects/report/test_report_with_gitleaks.py#L147-L154), for example, was much simpler to write with the `create_jira_issue` method was extracted.

Reply:
@amp-rh Okay, that's fine then. Can you add a comment there explaining this? Just want to make sure anyone digging into this in the future understands.